src/main/scala/nl.knaw.dans.easy.solr4files/package.scala

```diff
@@ -176,12 +176,10 @@ package object solr4files extends DebugEnhancedLogging {
       if (left.getProtocol.toLowerCase == "file")
         Try(new File(left.getPath).length)
       else {
-        Try(http(left.toString).method("HEAD").asString).flatMap {
-          case response if response.isSuccess =>
-            response.headers("content-length").headOption
-              .map(cl => Success(cl.toLong))
-              .getOrElse(Failure(HttpStatusException(s"no content-length header found in response of $left", response)))
-          case response => Failure(HttpStatusException(s"getContentLength($left)", response))
+        val fileSizesUrl = left.toString.replace("bags", "bags/filesizes")
+        Try(http(fileSizesUrl).method("GET").asString).flatMap {
+          case response if response.isSuccess => Success(response.body.toLong)
```

Comment:
What if this body contains something that is not numeric? Please make it at least

```suggestion
          case response if response.isSuccess => Try { response.body.toLong }
```

Reply:
done